.github/workflows/ci.yml

```diff
@@ -216,8 +217,8 @@ jobs:
       - name: Cache
         run: |
           docker exec pmm-managed-server go clean -testcache
+          docker exec --workdir=/root/go/src/github.com/percona/pmm-managed pmm-managed-server find . -type d -name fuzzdata -exec rm -r {} +
           rm -fr ~/.cache/go-build
-          mkdir -p ~/.cache
```

Comment:
why this is deleted? maybe there was some case when there is no cache dir and thus we need to create it ?

Reply:
It was not present in the same step in other job. These cache steps are identical, so I can't find a reason why we need it only here.

Reply:
@BupycHuk maybe you know?

Reply:
it's bit different from other repos, because we use docker here, that's why it is required.

Reply:
but looks like it's not required